### DISCUSSION TITLE
Unfortunately, we were unable to verify your academic affiliation based on the information you submitted on May 18, 2023 for @FSWei.

### DISCUSSION THREAD

**Discussion by FSWei** [community/community#55874] [GitHub Education]
### Hi everyone,

Hello,I am a Chinese high school student.
On February 1st of this year, I successfully applied for the Github student discount👇
![image](https://github.com/community/community/assets/95010451/88c8ad0e-0ba4-47eb-93a3-17f555e8ce6d)
But today, I suddenly received an email requesting me to re authenticate my student identity within 30 days, otherwise my student discount will be cancelled👇
![image](https://github.com/community/community/assets/95010451/0d88e7f2-57af-45a0-b5d9-cdf7b76e3c51)
The most frustrating thing is that I provided even more comprehensive information than before, but kept prompting "Unfortunately, we were unable to verify your academic affiliation based on the information you submitted on May 18, 2023"😭😭😭
I can guarantee that I have fully taken the photos as required, but it still cannot be done. If possible, I hope that human customer service can provide me with assistance.
I have tried 30 times, almost two hours😭
![image](https://github.com/community/community/assets/95010451/63c8dba5-5d97-4f82-a48a-cfc82067077c)

May I ask what I should do? I'm really about to cry😭😭😭

**Comment by ryanlua** [comment]
Hi @FSWei,

You shouldn't spam applications to GitHub Education as they can just blacklist you if you abuse the service. As for your application, check the list below with common problems that could get denied. 

- **Check if your school is recognized by GitHub** Check to see if your school is in [GitHub Campus](https://education.github.com/schools) or [Microsoft Education Program](https://www.microsoft.com/en-in/education). If not, then you may need to contact GitHub Support for additional support.
- **Use up to date documents/information.** Provide the most up to date information and documents otherwise you might be rejected if it is too outdated.
- **Ensure your GitHub profile matches.** Make sure all information such as your name or location both match your profile and documents.
- **Don't use a VPN.** GitHub may detect you using a VPN and automatically deny you access. Try applying without any VPN.
- **Translate your documents.** If your school is in another language than English than you should provide both the original untranslated document and English translated document.

You can read "[Why wasn't my application to Global Campus for students approved?](https://docs.github.com/en/education/explore-the-benefits-of-teaching-and-learning-with-github-education/github-global-campus-for-students/why-wasnt-my-application-to-global-campus-for-students-approved)" and the "[Student Developer Pack Application & FAQs ➡️ Start Here #17814](https://github.com/orgs/community/discussions/17814)" discussion for more information about Global Campus rejections. 

As a reminder you do not need to have a Student ID to get the GitHub Student Developer Pack, for more information see "[How to get the GitHub Student Developer Pack without a student ID](https://github.blog/2019-07-30-how-to-get-the-github-student-developer-pack-without-a-student-id/)"

If none of this helps, you can contact GitHub for support at [Get help with GitHub Education](https://support.github.com/contact/education).

Update me if you fix your issue or you followed the directions and still get rejected. I highly advise you to not mess with GitHub Education by spamming applications.

**Reply by FSWei** [reply]
Sorry, I'm not sure what you mean by spam, but I haven't sent any emails to Github's email. The emails in the screenshot are all feedback emails from Github after I applied for student discounts. The information I provided when applying for the student discount fully meets the requirements you mentioned, and my application was approved a few months ago, but today it cannot be approved. I am very confused

**Reply by FSWei** [reply]
When I access Get Help with GitHub Education, it prompts "No longer provides separate manual reviews and status updates"😭

**Reply by ryanlua** [reply]
> When I access Get Help with GitHub Education, it prompts "No longer provides separate manual reviews and status updates"😭

You can get around it by selecting “Other” option when submitting a GitHub support ticket. 

First though try seeing if this discussion can help you. Can you state what documents you have submitted to GitHub? Examples: Letter of admission, transcripts, etc.

**Reply by FSWei** [reply]
Thank you. I have tried submitting a GitHub support ticket, but I have not received a response yet. The document I have provided is my student ID card, which includes my school, name, photo, student ID, class, enrollment date, and expiration date.

**Reply by FSWei** [reply]
I just tried again three times and got the same result😭
![image](https://github.com/community/community/assets/95010451/bd72786e-b289-4603-a67d-f3949e51efa7)

**Reply by ryanlua** [reply]
> Thank you. I have tried submitting a GitHub support ticket, but I have not received a response yet. The document I have provided is my student ID card, which includes my school, name, photo, student ID, class, enrollment date, and expiration date.

Can you try to get an official or unofficial transcript from your admissions office and try using that?

**Reply by FSWei** [reply]
Thank you. I have tried to upload my transcript (including my name, student ID, class, major, ranking, and credits), but I still have not passed the application. And I haven't received any response on GitHub Support.
![image](https://github.com/community/community/assets/95010451/0e1ee681-312d-4f6c-aa7a-e9863c2e5cf9)

**Reply by FSWei** [reply]
When I select my email, it shows "1156337657@qq.com has already been used to obtain an academic discount on GitHub account(s) FSWei." below (because I have previously verified this account with this email and successfully). Does this have an impact? If it has an impact, should I delete this email and add it again?
![image](https://github.com/community/community/assets/95010451/214fca2d-c9bf-4250-9fe0-695e711c0529)

**Comment by yashwardhan-jyani** [comment]
Hey @FSWei ,
i am facing the same problem as you.
My application was approved a few months back but now it is being rejected, if you find something that works for you please tell me too.

**Comment by ajayvardhanreddy-purdue** [comment]
Hello team,

I wanted to follow up regarding my application, which has been getting rejected even after I submitted all the required documents, including my class schedule, I-20, and admission letter. I am unsure if there might be an issue with the documents or if additional information is needed.

Could you please assist me in identifying and resolving the problem? Thank you for your help.

Best regards,
Ajay.

**Comment by diegoprass** [comment]
É  Ruim o github não deixar fazer um simples upload de um documento, ter que tirar uma foto é algo muito ultrapassado. Estou com poblema semelhante, pois todos os meus documentos são digitais.
